lms/djangoapps/jabber/views.py

```diff
@@ -6,15 +6,16 @@
 from django.shortcuts import redirect
 from django_future.csrf import ensure_csrf_cookie
 
-from courseware.courses import get_course_with_access, \
-                               registered_for_course
+from courseware.courses import get_course_with_access,\
```

Comment:
We should avoid line continuation characters in Python.

This can be rewritten as 2 separate import statements.

```python
from courseware.courses import get_course_with_access
from courseware.courses import registered_for_course
```